site/content/docs/main/code-standards.md

```diff
@@ -8,6 +8,19 @@ toc: "true"
 
 When opening a pull request, please fill out the checklist supplied the template. This will help others properly categorize and review your pull request.
 
+### PR title
+
+Make sure that the pull request title summarizes the change made (and not just "fixes issue #xxxx"):
+
+Example PR titles:
+
+ - "Check for nil when validating foo"
+ - "Issue #1234: Check for nil when validating foo"
+
+### Cherry-pick PRs
+
+When a PR to main needs to be cherry-picked to a release branch, please wait until the main PR is merged first before creating the CP PR. If the CP PR is made before the main PR is merged, there is a risk that PR modifications in response to review comments will not make it into the CP PR.
+
```

Comment:
Should add some guideline regarding title for cherry-pick PRs.

Reply:
Should we perhaps have some automation in form of github actions which can do the manual work of Cherry Pick while also conforming to the title guidelines?

Reply:
Like kubernetes tide /cherry-pick? would be cool.

Reply:
https://prow.k8s.io/command-help#:~:text=/cherrypick%20%5Bbranch%5D

Reply:
@anshulahuja98 @kaovilai Having the ability to cherry-pick via bot would add some options. Note that this sort of automated cherry-pick might require an additional github action (or manual intervention) if there is a changelog file, since the CP PR will have a new number, so the changelog file will need to be renamed. I think that sort of automation is probably out of scope for this PR though and should be considered as a follow-on action/investigation.

Reply:
@kaovilai well changelog filename check but also actually changing the filename, since cherry-picking the commit will have the wrong filename. When doing this manually, after cherry-picking the commit and creating the new PR you have to go in and change the filename to match the new PR number and amend the commit.

Reply:
> changing the filename

github actions can do that automatically :)

Reply:
It will need to do it *after* creating the PR, though, since we don't know what the filename will be until the PR exists.